Fix-Website/app/app.js

```diff
@@ -34,25 +32,44 @@
     app.directive("infoZone", function () {
         return {
             restrict: 'E',
-            templateUrl: "Templates/info.html",
-            controller: function () {
-                this.form = false;
-                this.clickProduct = function () {
-
-                    this.form = !this.form;
-                };
-            },
-            controllerAs: 'info'
+            templateUrl: "Templates/info.txt"
         };
     });
 
-    app.controller('ReferenceController', ['$scope', function ($scope) {
+    /**
+     * Controller for setting up the scope.
+     */
+    app.controller('ReferenceController', ['$scope','$http', function ($scope, $http) {
```

Comment:
IIRC, you need to declare the values up front, since otherwise Angular won't know what to inject, right? Since this isn't going through the Closure compiler, there are some things we don't get for free. The dependency injection bit is part of that. Maybe?